### PR TITLE
Travis CI fixes

### LINKED ISSUE #69: Investigate issues with Travis CI and Minio
Currently we are unable to test the data transformers with Minio in Travis CI, due to a problem with running the minio container in travis. The container appears to crash with this error:

``` JSON
time="2016-06-04T13:56:23Z" level=fatal msg="Unable to colorize experimental update notification string ‘’." error="inappropriate ioctl for device" probe="{\"cause\":25,\"trace\":[{\"line\":50,\"file\":\"notifier.go\",\"func\":\"main.colorizeUpdateMessage\"},{\"line\":93,\"file\":\"update-main.go\",\"func\":\"main.updateMessage.String\",\"env\":{\"Tags\":[\"\"]}}],\"sysinfo\":{\"host.arch\":\"amd64\",\"host.cpus\":\"2\",\"host.lang\":\"go1.6\",\"host.name\":\"a20ec40f382b\",\"host.os\":\"linux\",\"mem.heap.total\":\"4.8MB\",\"mem.heap.used\":\"2.0MB\",\"mem.total\":\"6.3MB\",\"mem.used\":\"2.0MB\"}}" 
```

Investigate why this happens and how to circumvent it, or wait for a minio update to fix this.

Once solved, we should rollback: https://github.com/Cerfoglg/data-transformers/commit/f2e674a2c59fda79e1a207edcb25f5c044d25e15 and the same fixes in https://github.com/Cerfoglg/data-transformers/commit/f2e674a2c59fda79e1a207edcb25f5c044d25e15

### PR DESCRIPTION
**Pull request by Cerfoglg**
Fixes #69
<a href='#crh-start'></a><a href='#crh-data-%7B%22processed%22%3A%20%5B%22https%3A//github.com/benchflow/data-transformers/pull/73%23discussion_r69300545%22%2C%20%22https%3A//github.com/benchflow/data-transformers/pull/73%23discussion_r69300995%22%2C%20%22https%3A//github.com/benchflow/data-transformers/pull/73%23discussion_r69306897%22%2C%20%22https%3A//github.com/benchflow/data-transformers/pull/73%23discussion_r69306906%22%2C%20%22https%3A//github.com/benchflow/data-transformers/pull/73%23discussion_r69309687%22%2C%20%22https%3A//github.com/benchflow/data-transformers/pull/73%23discussion_r69309688%22%5D%2C%20%22comments%22%3A%20%7B%22Pull%20cedb103257389f8bce7fecca701bdf39537aab99%20.travis.yml%2030%22%3A%20%7B%22html_url%22%3A%20%22https%3A//github.com/benchflow/data-transformers/pull/73%23discussion_r69300995%22%2C%20%22comments%22%3A%20%5B%7B%22body%22%3A%20%22Why%20do%20you%20run%20the%20same%20command%20twice%3F%22%2C%20%22created_at%22%3A%20%222016-07-01T13%3A42%3A56Z%22%2C%20%22user%22%3A%20%7B%22avatar_url%22%3A%20%22https%3A//avatars.githubusercontent.com/u/7163641%3Fv%3D3%22%2C%20%22html_url%22%3A%20%22https%3A//github.com/VincenzoFerme%22%7D%7D%2C%20%7B%22body%22%3A%20%22%40VincenzoFerme%20My%20bad%2C%20fixed%22%2C%20%22created_at%22%3A%20%222016-07-01T14%3A24%3A19Z%22%2C%20%22user%22%3A%20%7B%22avatar_url%22%3A%20%22https%3A//avatars.githubusercontent.com/u/8819678%3Fv%3D3%22%2C%20%22html_url%22%3A%20%22https%3A//github.com/Cerfoglg%22%7D%7D%2C%20%7B%22body%22%3A%20%22%3A%2B1%3A%3Ca%20href%3D%27%23crh-update-none%27%3E%3C/a%3E%22%2C%20%22created_at%22%3A%20%222016-07-01T14%3A42%3A59Z%22%2C%20%22user%22%3A%20%7B%22avatar_url%22%3A%20%22https%3A//avatars.githubusercontent.com/u/7163641%3Fv%3D3%22%2C%20%22html_url%22%3A%20%22https%3A//github.com/VincenzoFerme%22%7D%7D%5D%2C%20%22title%22%3A%20%22File%3A%20.travis.yml%3AL18-35%22%7D%2C%20%22Pull%20cedb103257389f8bce7fecca701bdf39537aab99%20.travis.yml%204%22%3A%20%7B%22html_url%22%3A%20%22https%3A//github.com/benchflow/data-transformers/pull/73%23discussion_r69300545%22%2C%20%22comments%22%3A%20%5B%7B%22body%22%3A%20%22Remove%22%2C%20%22created_at%22%3A%20%222016-07-01T13%3A39%3A44Z%22%2C%20%22user%22%3A%20%7B%22avatar_url%22%3A%20%22https%3A//avatars.githubusercontent.com/u/7163641%3Fv%3D3%22%2C%20%22html_url%22%3A%20%22https%3A//github.com/VincenzoFerme%22%7D%7D%2C%20%7B%22body%22%3A%20%22%40VincenzoFerme%20My%20bad%2C%20fixed%22%2C%20%22created_at%22%3A%20%222016-07-01T14%3A24%3A23Z%22%2C%20%22user%22%3A%20%7B%22avatar_url%22%3A%20%22https%3A//avatars.githubusercontent.com/u/8819678%3Fv%3D3%22%2C%20%22html_url%22%3A%20%22https%3A//github.com/Cerfoglg%22%7D%7D%2C%20%7B%22body%22%3A%20%22%3A%2B1%3A%3Ca%20href%3D%27%23crh-update-complete%27%3E%3C/a%3E%22%2C%20%22created_at%22%3A%20%222016-07-01T14%3A43%3A00Z%22%2C%20%22user%22%3A%20%7B%22avatar_url%22%3A%20%22https%3A//avatars.githubusercontent.com/u/7163641%3Fv%3D3%22%2C%20%22html_url%22%3A%20%22https%3A//github.com/VincenzoFerme%22%7D%7D%5D%2C%20%22title%22%3A%20%22File%3A%20.travis.yml%3AL9-16%22%7D%7D%7D'></a>
<a href='https://www.codereviewhub.com/'></a>
- [x] <a href='#crh-comment-Pull cedb103257389f8bce7fecca701bdf39537aab99 .travis.yml 4'></a> &nbsp;<b><a href='https://github.com/benchflow/data-transformers/pull/73#discussion_r69300545'>File: .travis.yml:L9-16</a></b>
- <a href='https://github.com/VincenzoFerme'></a> Remove
- <a href='https://github.com/Cerfoglg'></a> @VincenzoFerme My bad, fixed
- [x] <a href='#crh-comment-Pull cedb103257389f8bce7fecca701bdf39537aab99 .travis.yml 30'></a> &nbsp;<b><a href='https://github.com/benchflow/data-transformers/pull/73#discussion_r69300995'>File: .travis.yml:L18-35</a></b>
- <a href='https://github.com/VincenzoFerme'></a> Why do you run the same command twice?
- <a href='https://github.com/Cerfoglg'></a> @VincenzoFerme My bad, fixed

<a href='https://www.codereviewhub.com/benchflow/data-transformers/pull/73?mark_as_completed=1'></a>&nbsp;<a href='https://www.codereviewhub.com/benchflow/data-transformers/pull/73?approve=1'></a>&nbsp;<a href='https://github.com/benchflow/data-transformers/pull/73'></a>
<a href='#crh-end'></a>

### CODE REVIEW

**Comment by VincenzoFerme** [comment]
This pull request is now managed with CodeReviewHub. <a href='https://github.com/benchflow/data-transformers/pull/73'>Refresh</a>.<a href='#crh-update-none'></a>

File: .travis.yml [hunk 1]
```diff
@@ -9,6 +9,7 @@ language: bash
 branches:
   only:
     - dev
+    - travis-fixes
```
Comment (VincenzoFerme) [inline .travis.yml]: Remove

File: .travis.yml [hunk 2]
```diff
@@ -17,15 +18,17 @@ before_install:
...
+  - docker run --name minio -e "MINIO_ACCESS_KEY=AKIAIOSFODNN7EXAMPLE" -e "MINIO_SECRET_KEY=wJalrXUtnFEMI/K7MDENG/bPxRfiCYEXAMPLEKEY" -d minio/minio /export
+  - sleep 60
+  - docker cp ./test/data/runs minio:/export/runs
+  - docker run -d --name cassandra cassandra:3.0
+  - sleep 60
+  - docker run -d --name cassandra2 --link cassandra:cassandra cassandra:3.0
+  - sleep 60
+  - docker cp ./test/data/benchflow.cql cassandra:/
+  - docker exec cassandra cqlsh -f /benchflow.cql
+  - docker run --rm --name spark --link cassandra:cassandra --link minio:minio --entrypoint=/test/runTests.sh sparktests
```
Comment (VincenzoFerme) [inline .travis.yml]: Why do you run the same command twice?

Comment (Cerfoglg) [inline .travis.yml]: @VincenzoFerme My bad, fixed

File: .travis.yml (hunk 1 above)
Comment (Cerfoglg) [inline .travis.yml]: @VincenzoFerme My bad, fixed

File: .travis.yml (hunk 2 above)
Comment (VincenzoFerme) [inline .travis.yml]: :+1:<a href='#crh-update-none'></a>

File: .travis.yml (hunk 1 above)
Comment (VincenzoFerme) [inline .travis.yml]: :+1:<a href='#crh-update-complete'></a>

**Pull request merged**